src/pages/RequestCallPage.js

```diff
@@ -102,19 +99,23 @@ class RequestCallPage extends Component {
             return;
         }
 
-        const personalPolicy = _.find(this.props.policies, policy => policy && policy.type === CONST.POLICY.TYPE.PERSONAL);
-        if (!personalPolicy) {
+        // If there's a lastAccessedWorkspace in Onyx then use that as the policy for the call, otherwise use the personal policy
```

Comment:
This comment can probably be removed or improved. It describes what the code is doing but would be better if it could perhaps explain **why** a `policyID` is required to request an inbox call or **why** we'd prefer the workspace over the personal policy.

Reply:
I removed it. With the updated way I'm setting this, I don't think a comment is necessary.